src/viam/ur/module/ur_arm.cpp

```diff
@@ -496,8 +496,8 @@ URArm::KinematicsData URArm::get_kinematics(const ProtoStruct&) {
     const std::streamsize fileSize = sva_file.tellg();
     sva_file.seekg(0, std::ios::beg);
 
-    // Create a buffer to hold the file contents
-    std::vector<unsigned char> kinematics_bytes(fileSize);
+    // Create a buffer to hold the file contents, int long fits in unsigned long
+    std::vector<unsigned char> kinematics_bytes((unsigned long)fileSize);
```

Comment:
Don't use C-style casts. Use `static_cast` for that. Also, I think it is worth thinking a bit more about what this warning was telling us. If `streamsize` can be negative, I don't think you want to cast it to positive and then allocate that many bytes.